packages/react-native-editor/ios/Podfile.lock

```diff
@@ -556,13 +556,13 @@ SPEC CHECKSUMS:
   React-jsiexecutor: a27badbbdbc0ff781813370736a2d1c7261181d4
   React-jsinspector: 8a3d3f5dcd23a91e8c80b1bf0e96902cd1dca999
   React-logger: 1088859f145b8f6dd0d3ed051a647ef0e3e80fad
-  react-native-blur: 8cd9b4a8007166ad643f4dff914c3fddd2ff5b9a
+  react-native-blur: 3e9c8e8e9f7d17fa1b94e1a0ae9fd816675f5382
   react-native-get-random-values: b6fb85e7169b9822976793e467458c151c3e8b69
   react-native-safe-area: c9cf765aa2dd96159476a99633e7d462ce5bb94f
-  react-native-safe-area-context: e471852c5ed67eea4b10c5d9d43c1cebae3b231d
+  react-native-safe-area-context: f0906bf8bc9835ac9a9d3f97e8bde2a997d8da79
   react-native-slider: dff0d8a46f368a8d1bacd8638570d75b9b0be400
-  react-native-video: afb806880af4f6612683ab678a793ae41bc39705
-  react-native-webview: e3b659a6d614bb37fb12a2de82c91a378c59d84b
+  react-native-video: 6dee623307ed9d04d1be2de87494f9a0fa2041d1
+  react-native-webview: 9f111dfbcfc826084d6c507f569e5e03342ee1c1
```

Comment:
Seems something changed in the `podspecs` of these Pods in the last release ([reference](https://github.com/WordPress/gutenberg/commit/2d4c4770215fcff72d7ad71b211fedb00de880ed#diff-ce3a290cf4efc62a6f6097db3b4f49ba4799ca7c66f599bc16d94ed93586028c)). Since we're actively working on the RN ugprade, I'm wondering if that affected somehow the pod installation when cutting the betafix `1.98.1` 🤔. In any case, in this PR we are reverting back to the values we previously had.

Reply:
Thanks for the heads up, since this affects only the demo app we should be ok with the main iOS host app 👍